pkg/wireguard/conf.go

```diff
@@ -95,6 +95,28 @@ func (e *Endpoint) String() string {
 	return dnsOrIP + ":" + strconv.FormatUint(uint64(e.Port), 10)
 }
 
+// Equal compare to endpoints
+func (e *Endpoint) Equal(b *Endpoint) bool {
```

Comment:
Nice :+1:  making this reusable is great

Reply:
I did also replace the code in mesh to use it, but when I reviewed my commits to prepare this PR, I saw that the logic there was different: It checks the DNS name first ... so I left it as is... but maybe it should be unified too. I didn't want to introduce changes there for now, so we can discuss it here, or we can postpone it and I'll rethink it an do another MR.

Reply:
Yes, I guess we could add a parameter that specifies what to check first, but this doesn't seem to important for this PR

Reply:
well I did it ;)